src/test/java/com/dabsquared/gitlabjenkins/util/CommitStatusUpdaterTest.java

```diff
@@ -82,7 +91,8 @@ public void setUp() throws Exception {
 	    when(build.getCauses()).thenReturn(new ArrayList<Cause>(Collections.singletonList(upCauseLevel1)));
 	    when(upCauseLevel1.getUpstreamCauses()).thenReturn(new ArrayList<Cause>(Collections.singletonList(upCauseLevel2)));
 	    when(upCauseLevel2.getUpstreamCauses()).thenReturn(new ArrayList<Cause>(Collections.singletonList(gitlabCause)));
-	    
+	    when(taskListener.getLogger()).thenReturn(new PrintStream("/dev/null"));
```

Comment:
This fails on Windows because there is no /dev/null  Based on https://stackoverflow.com/questions/313111/is-there-a-dev-null-on-windows I believe you can reference 'nul' on Windows systems. I'm not sure offhand how this test could check if it is running on Windows, but I think there must be some simple way. As part of the plugin validation process it is tested on Linux and Windows machines, see https://ci.jenkins.io/blue/organizations/jenkins/Plugins%2Fgitlab-plugin/detail/PR-747/6/pipeline  Can you try to fix this?

Reply:
sure, no problem, thanks for details.